### ISSUE TITLE
Loss problem

### ISSUE THREAD

**Issue by zmj2** [JusperLee/DANet-For-Speech-Separation#2]
你好，想问一下关于loss的问题解决了吗。我的期末作业是实现语音分离，我想选这个模型，但是它训练的时候loss确实不怎么变，不知道是什么问题。
我看在loss上原文是这么写的：
loss = mixture.expand(mixture.size(0), mixture.size(1), wfm.size(2)) * (wfm - estimated_mask)
loss = loss.view(-1, loss.size(1)*loss.size(2))
return torch.mean(torch.sum(torch.pow(loss, 2), 1))
这样会报错，因为mixture本来的第三维是F=149。然后我猜他可能的意思是将mixture原本的维度(B, T, F)先变成(B, T*F, 1)，然后再将第三维expand到两个，但是我按照这样改了以后还是不行。
